pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
The proxier needs to have the information to write the correct iptables rules. So we have to expose this in the API so that it's transmitted to the minion.